PublicationsRouter/v2/api/MatchingConfig.md

```diff
@@ -0,0 +1,42 @@
+# Matching Config Structure
+Document contains structure of matching configuration JSON for both submission and retrieval. Used in POST/GET requests respectively against endpoint `/config`.
+
```

Comment:
I suggest the GET is presented before the POST, partly because then you need only include one sample of dummy data.